link/kprobe.go

```diff
@@ -335,7 +343,7 @@ func tracefsProbe(typ probeType, args probeArgs) (*perfEvent, error) {
 	}
 
 	// Get the newly-created trace event's id.
-	tid, err := getTraceEventID(group, args.symbol)
+	tid, err := getTraceEventID(group, sanitizedSymbol(args.symbol))
```

Comment:
Is this still necessary with the fallback to tracefs? I'm a bit foggy on this, but doesn't this modify the symbol name in a way that it no longer matches the kernel's? Couldn't we retry a PMU probe with a sanitized symbol in that case?

Reply:
Yes, on line L396, we have to sanitize the `EVENT` part of kprobe_event.

Reply:
Okay, so this means the 'name' part of the event. That makes sense.